Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -42,7 +42,11 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
        .. code-block:: shell-session
 
            export NAME="$(whoami)-$RANDOM"
-           gcloud container clusters create "${NAME}" --zone us-west2-a 
+           # Create the node pool with the following taint to guarantee that
+           # Pods are only scheduled in the node when Cilium is ready.
```

Comment:
The `#` in these comments will be interpreted as prompts. You could prefix most lines with `$` to fix this.